src/tasks/templates/migration-file.js

```diff
@@ -0,0 +1,15 @@
+module.exports = `module.exports = function (block) {
+  // Example to change a string to boolean
+  // block.{{ fieldname }} = !!(blok.{{ fieldname }})
+
+  // Example to transfer content from other field
+  // block.{{ fieldname }} = block.other_field
+
+  // Example to transform a markdown field into a richtext field
+  // const { MarkdownParser } = require('prosemirror-markdown')
```

Comment:
That brings up the question. How the user can use npm plugins that are not globally installed?

Reply:
He will need to install the package locally inside migrations folder. The node js module resolution will get the packages first in this folder, before to try to get the package globally.